src/Toolbar/ToolbarItem.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ElementType, FC, HTMLAttributes, MouseEvent } from 'react';
+import classNames from 'classnames';
+import { Icon } from '../Icon/Icon';
+
+type HTMLElementWithDisabled<T> = Partial<T> & { disabled?: boolean };
+
+export interface ToolbarItemProps
+  extends HTMLElementWithDisabled<HTMLAttributes<HTMLElement>> {
+  tag?: ElementType;
+  classname?: string;
+  active?: boolean;
+  url?: string;
+  label?: string;
+  srText?: string;
+  iconName?: string;
+  isIconSmall?: boolean;
+  alert?: boolean;
+  badge?: number;
+  onLinkClick?: (event: MouseEvent<HTMLAnchorElement>) => void;
+}
+
+export const ToolbarItem: FC<ToolbarItemProps> = ({
+  active = false,
+  badge,
+  url,
+  srText,
+  iconName = 'it-comment',
+  isIconSmall,
```

Comment:
To be honest I feel this one should come from the `Toolbar` context rather than as a prop. I cannot find an instance of `icon-sm` outside the small version of the toolbar. But do not worry for now, will think about it later.

Reply:
Yes I feel so. If we add class 'toolbar-small' to Toolbar component, icons in ToolbarItem should be small too. That was my first intention but I couldn't find the way to do it for now.

Reply:
Don’t worry for now, it can be a follow up of this PR

Reply:
Removed, but now the ToolbarItem component lacks the 'small' size. In the EsempioSmall the icons are exactly the same size as they are in Esempio or EsempioMedium.